trade-finance-manager-api/src/utils/date.test.js

```diff
@@ -53,4 +53,22 @@ describe('utils - date', () => {
       expect(isValidIsoMonth('2023-11')).toBe(true);
     });
   });
+
+  describe('getIsoStringWithOffset', () => {
+    it('returns an ISO-8601 string with offset when timezone is GMT', () => {
+      const date = new Date('2024-03-02');
+
+      const result = getIsoStringWithOffset(date);
+
+      expect(result).toBe('2024-03-02T00:00:00+00:00')
+    });
+
+    it('returns an ISO-8601 string with offset when timezone is BST', () => {
+      const date = new Date('2024-07-02');
+
+      const result = getIsoStringWithOffset(date);
+
+      expect(result).toBe('2024-07-02T01:00:00+01:00')
```

Comment:
Can I check that existing implimentation did this too? I don't want to introduce timezones around the place where it might not be properly handled

Reply:
Yep, previous was `moment().format()` which returns BST
```
moment('2024-07-01').format() // 2024-07-01T00:00:00+01:00
```

Reply:
Well thats surprising. Okay, thanks!